lib/pipeline.js

```diff
@@ -499,6 +487,50 @@ class PipelineModel extends BaseModel {
         return admin;
     }
 
+    /**
+     * This function deletes admins who does not have proper
+     * GitHub permission, and returns an proper admin.
+     * @method getFirstAdmin
+     * @return {Promise}
+     */
+    async getFirstAdmin() {
+        /* eslint-disable no-restricted-syntax */
+        for (const username of Object.keys(this.admins)) {
+            try {
+                /* eslint-disable-next-line no-await-in-loop */
```

Comment:
should be a `//` format comment for next line only
like https://github.com/screwdriver-cd/models/blob/master/lib/templateFactory.js#L65

Reply:
I fixed. Thank you!